### ISSUE TITLE
World should magically change, what players see

### ISSUE THREAD

**Issue by Feinbube** [oaken-source/noobcraft#2]
Like in cow, all players should see the old state.

If "sight radius" is going to make it into the game, the world deliver information accordingly.

**Comment by oaken-source** [comment]
true. also, changes made to the world state - beyond orders given to units - should be contained.
